src/fortran_routing/Reservoir_Binding/kernels/test_compute_kernel.py

```diff
@@ -0,0 +1,239 @@
+"""
+import pyximport
+pyximport.install()
+import sys
+#Monkey patch pximport to not build modules not under test
+
+finder = next(finder
+              for finder in sys.meta_path
+              if isinstance(finder, pyximport.PyxImporter))
+find_module_original = finder.find_module
+
+def find_module_new(self, fullname, package_path=None):
+     if fullname.startswith('compute_kernel'):
+         return find_module_original(fullname, package_path)
+
+finder.find_module = find_module_new.__get__(finder, pyximport.PyxImporter)
+"""
+import pytest
```

Comment:
@jdmattern-noaa 
Could we set up a unit-test that validates execution in a thread-pool?

We'll want to check:
1) That it produces the same result as sequential calculation for the same reservoirs
2) That it actually drops the interpreter lock and allows simultaneous computation

Reply:
### Build/Install
```
cd src/fortran_routing/Reservoir_Binding/kernels/fortran/Reservoir
export NETCDF_LIB=/usr/lib64/openmpi/lib/
export NETCDF_INC=/usr/include/openmpi-x86_64
make
cd src/fortran_routing/Reservoir_Binding/kernels
python setup.py install
```
### Test: 
```
python -m pytest test_compute_kernel.py
==================================================================== test session starts =====================================================================
platform linux -- Python 3.6.8, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /home/james.halgren/git/t-route-jsh_blank/src/fortran_routing/Reservoir_Binding/kernels
plugins: cov-2.8.1
collected 6 items

test_compute_kernel.py ......                                                                                                                          [100%]

===================================================================== 6 passed in 0.21s ======================================================================
```

Reply:
I will update the instructions with those exports.